lib/index.js

```diff
@@ -1,6 +1,10 @@
+
+require('dotenv').config();
```

Comment:
Yeah a lot of different things look for it. `dotenv` is just a npm module to parse the file and load them into your process.env